libs/utils/utils.js

```diff
@@ -803,7 +803,8 @@ function decorateMeta() {
     if (meta.getAttribute('property') === 'hlx:proxyUrl') return;
     try {
       const url = new URL(meta.content);
-      meta.setAttribute('content', `${origin}${url.pathname}${url.search}${url.hash}`);
+      const localizedPath = localizeLink(`${origin}${url.pathname}`);
```

Comment:
Is it certain that this always returns a path? `localizeLink` can return the whole URL in certain scenarios. If this happens, then the next line, `${origin}${localizedPath}...`, might be in an odd format.

Reply:
@overmyheadandbody I think because full paths are typically utilized in metadata.xlsx and the metadata block, I do not believe this will be an issue, but do you think it would be preferable to add a check like this:
```
const localizedURL = localizeLink(`${origin}${url.pathname}`);
const localizedPath = localizedURL.includes(origin) ? localizedURL : `${origin}${localizedURL}`;
```

Reply:
That's what I would personally go for, yes. But if you weren't able to find any such use-cases, I'm fine with having it as-is, I'm keeping my approval of the PR 😄 

If you do decide to implement the suggestion, though, I'd choose slightly different variable naming, as `localizedPath` would actually be `origin` + `pathname`, so not just a path. And of course, next line can just use this new constant rather than concatenating `${origin}${localizedPath}`

Reply:
Done!